clojure-mode.el

```diff
@@ -955,7 +955,13 @@ highlighted region)."
                              (setq docelt (1- docelt)))))
                        (and (zerop docelt) (<= (point) startpos)
                             (progn (forward-comment (point-max)) t)
-                            (= (point) (nth 8 state)))))
+                            (= (point) (nth 8 state))))
+                     ;; In a def, at last position is not a docstring
+                     (not (and (string= "def" firstsym)
+                               (save-excursion
+                                 (goto-char startpos)
```

Comment:
Is this necessary given the `goto-char` below?

Reply:
The first `goto-char` looks redundant but I think it is not, because there is a `(sexp-at-point)` below, and it needs the point at the correct position beforehand.

Also, the `(sexp-at-point)` form does not take any arguments so I cannot pass `startpos` to it.

Anyway, I tried without it and it didn't work. Happy to remove it if we find the way to make it work.

Reply:
Ah, great point. I didn't think of that! Yeah, we definitely need it because of the `(sexp-at-point)`.

Reply:
Go ahead and squash if you like. I will merge once I have the rights to do so.